src/App.css

```diff
@@ -21,11 +21,64 @@
     position: absolute;
     top:  6vw;
     left: 30vw;
-    width: 160px;
-    height: 160px;
+    width:  12vw;
+    height: 12vw;
     z-index: 10;
     background-repeat: no-repeat;
     background-size: cover;
     
 }
 
+.ButtonContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 2vw;
+    position: absolute;
+    right: 2vw;
+    
+}
+
+.top{
+    top: 2vw;
+}
+.bottom {
+    top: 12vw;
+}
+.App-Data{
+    position: absolute;
+    top: 28vw;
+    left: 30vw;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.no-data{
+    color: crimson;
+    font-style: italic;
+}
+
+.data-container {
+    position: absolute;
+    background-color: #222222b0;
+    margin: 10px 0;
+    padding:  10px;
+    border-radius: 5px;
+    color: #fff ;
```

Comment:
Extra space at the end

Reply:
Also colors are usually put into variables, instead of the code directly.

[How to declare vars](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#declaring_custom_properties)

[How to use vars](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#referencing_custom_properties_with_var)